.github/workflows/ci.yml

```diff
@@ -67,11 +135,41 @@ jobs:
       resolution: "lowest-direct"
 
     steps:
-      - uses: actions/checkout@v3
-      - name: APT update
-        run: sudo apt-get update
-      - name: Install system dependencies
-        run: sudo apt-get install libgeos-dev libproj-dev
+      - name: Checkout library code
```

Comment:
is it not possible to reuse the above commands? the only difference is the dependency resolution right?

Reply:
Wüsste nicht wie, aber bin für Tipps offen

Reply:
github supports reusable workflows and composite actions, but I guess that's for another task. https://docs.github.com/en/actions/sharing-automations/avoiding-duplication

Reply:
added as task